proto/ardupilot_gazebo/msgs/motor_stats.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package ardupilot_gazebo.msgs;
+
+// Message to hold the status of a single motor.
+message MotorStats {
```

Comment:
```suggestion
message MotorStatus {
```

`MotorStatus` better aligns with the DroneCAN message names (`uavcan.equipment.esc.Status` and `uavcan.equipment.esc.StatusExtended`). See for example: https://github.com/dronecan/DSDL/blob/master/uavcan/equipment/esc/1034.Status.uavcan